pontoon/settings/base.py

```diff
@@ -789,10 +790,11 @@ def _default_from_email():
     "django.contrib.staticfiles.finders.FileSystemFinder",
     "django.contrib.staticfiles.finders.AppDirectoriesFinder",
 )
-STATICFILES_DIRS = [
+
+STATICFILES_DIRS = (
     os.path.join(TRANSLATE_DIR, "dist"),
     os.path.join(TRANSLATE_DIR, "public"),
-]
+)
```

Comment:
Nit: I'd leave that as it was, since it's not introducing any meaningful change.